Dockerfile

```diff
@@ -1,33 +1,12 @@
 FROM golang:alpine AS builder
 WORKDIR /src
 
-ENV USER=vpn-webauth
-ENV UID=10001 
-
 COPY go.mod .
 COPY go.sum .
 RUN go mod download
 
 RUN apk --update --upgrade --no-cache add git gcc g++ ca-certificates && update-ca-certificates
-RUN adduser \    
-    --disabled-password \    
-    --gecos "" \    
-    --home "/nonexistent" \    
-    --shell "/sbin/nologin" \    
-    --no-create-home \    
-    --uid "${UID}" \    
-    "${USER}"
 
 COPY . .
-RUN CGO_ENABLED=1 GOOS=linux go build -a -installsuffix cgo -ldflags="-w -s" -o /out/vpn-webauth .
-
-
-FROM alpine AS bin
-COPY --from=builder /out/vpn-webauth /
-COPY --from=builder /etc/passwd /etc/passwd
-COPY --from=builder /etc/group /etc/group
-COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
-COPY --from=builder /src/templates /templates/
 
-USER vpn-webauth
-ENTRYPOINT ["/vpn-webauth"]
+ENTRYPOINT ["go", "run", "server.go", "main.go"]
```

Comment:
I'm not familiar with `go run`; what is the reason why you replaced the previous way of building the project with this?

Reply:
See previous comment, I edited the Docker container due to issues I was having with getting the templates to update. I'll try again and see if I can post what the issues were.

Reply:
Build is working now